indico/MaKaC/webinterface/rh/conferenceDisplay.py

```diff
@@ -493,6 +498,7 @@ class ContributionsFilterCrit(filters.FilterCriteria):
 
 class RHContributionList( RHConferenceBaseDisplay ):
     _uh = urlHandlers.UHContributionList
+    _MENU_LINK_TO_DISABLE = 'contributionList'
```

Comment:
What about `MENU_ITEM`? In the end this is about which item in the menu represents this RH.